src/Symfony/Component/Console/Completion/CompletionInput.php

```diff
@@ -109,6 +109,10 @@ public function bind(InputDefinition $definition): void
         // complete argument value
         $this->completionType = self::TYPE_ARGUMENT_VALUE;
 
+        // shell cannot interpret quotes
+        $trimQuotes = function ($value): string {
+            return trim($value, '\'"');
```

Comment:
Would be more accurate with this regex ([tests](https://regex101.com/r/90STp2/1)):
```suggestion
            return preg_replace('~(^[\'"]|[\'"]$)~', '', $value);
```

Reply:
Not sure we need to support quotes in argument value. The code would be like:
```php
    function trimQuotes(string $value): string
    {
        $length = strlen($value);
        if (0 === $length) {
            return $value;
        }
        if (!in_array($value[0], ["'", '"'])) {
            return $value;
        }
        if (1 === $length || $value[0] !== $value[$length-1] || '\\' === $value[$length-2]) {
            // Removes starting quote
            return substr($value, 1);
        }

        // Removes starting and ending quote
        return substr($value, 1, $length-2);
    }
```